app/domain/authentication/authn_jwt/orchestrate_authentication.rb

```diff
@@ -24,23 +25,24 @@ def call
       def validate_uri_based_parameters
         @validate_uri_based_parameters.call(
           authenticator_input: @authenticator_input,
-          enabled_authenticators: @enabled_authenticators
+          enabled_authenticators: @installed_authenticators_class.enabled_authenticators_str
         )
       end
 
       def authenticate_jwt
-        relevant_authenticator = authenticator_name
         @logger.info(LogMessages::Authentication::AuthnJwt::JwtAuthenticatorEntryPoint.new(relevant_authenticator))
 
-        jwt_authenticator_configuration = @configuration_factory.create_jwt_configuration(@authenticator_input)
+        jwt_authenticator_configuration = @configuration_factory.call(
+          authenticator_input: @authenticator_input
+        )
         @jwt_authenticator.call(
           jwt_configuration: jwt_authenticator_configuration,
           authenticator_input: @authenticator_input
         )
       end
 
-      def authenticator_name
-        @authenticator_input.authenticator_name
+      def relevant_authenticator
+        @relevant_authenticator ||= @authenticator_input.authenticator_name
       end
```

Comment:
i still don't understand why we need this. why don't we call `@authenticator_input.authenticator_name` directly? or at least call this method `authenticator_name` and not `relevant_authenticator`